website/blog/2023-04-24-backup-frequency.md

```diff
@@ -0,0 +1,127 @@
+---
+slug: how-often-should-you-run-microsoft-365-backups
+title: "How often should you run Microsoft 365 backups?"
+description: "On the ideal cadence for backups"
```

Comment:
Can you add a slightly longer description, please?

Reply:
extended description